### PR TITLE
Feature/style 5677 test money making features

### PR DESCRIPTION
**Pull request by Drathal**

### CODE REVIEW

File: features/util/siteCountryMap.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    german: 'de',
+    sweden: 'se',
+    netherlands: 'nl',
+    unitedStates: 'com',
```
Comment (peter-gimahhot) [inline features/util/siteCountryMap.js]: site is `us`, tld is `com`

File: features/util/events.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+const { defineSupportCode } = require('cucumber');
+const { client } = require('nightwatch-cucumber');
+
+defineSupportCode(({ registerHandler }) => {
+    registerHandler('BeforeFeatures', function(features, cb) {
+        console.log('Testing on', process.env.BASE_URL); // eslint-disable-line
+        console.log(''); // eslint-disable-line
```
Comment (Drathal) [inline features/util/events.js]: yep i want to print something to the console :)

File: features/util/siteCountryMap.js (hunk 1 above)
Comment (Drathal) [inline features/util/siteCountryMap.js]: yes but the  toplevel domain is " .com"

Comment (Drathal) [inline features/util/siteCountryMap.js]: but the naming is not straight ... thats true

File: features/page_objects/all.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    url
+    .replace(/(-de\.)/, `-${newLang}.`)
+    .replace(/(\.de$)/, `.${newLang}`);
+
+module.exports = {
+    url: function(path) {
+        const launchUrl = replaceCountry(this.api.launchUrl, siteCountryMap[data.lang] || 'de');
+        const launchPath = path ? path : '';
+        const sem = data.userType ? `sclid=${data.userType}` : '';
+        const url = `${launchUrl}${launchPath}?${sem}`;
```
Comment (Drathal) [inline features/page_objects/all.js]: yes but for routing its not important if there is a ? at the end...

**Pull request merged**